apps/stable_diffusion/src/models/model_wrappers.py

```diff
@@ -277,7 +279,7 @@ def forward(self, input):
             extra_args=get_opt_flags("vae", precision=self.precision),
             base_model_id=self.base_model_id,
         )
-        return shark_vae
```

Comment:
does this break functionality elsewhere? do we never expect only one output from these functions?

Reply:
In addition, I have tested txt2img and img2img with my patch.